cfme/utils/appliance/implementations/ui.py

```diff
@@ -571,7 +572,7 @@ def go(self, _tries=0, *args, **kwargs):
             self.prerequisite_view = self.prerequisite()
             try:
                 self.check_for_badness(self.step, _tries, nav_args, *args, **kwargs)
-            except (exceptions.CandidateNotFound, exceptions.ItemNotFound) as e:
+            except (exceptions.CandidateNotFound, exceptions.ItemNotFound, CandidateNotFound) as e:
```

Comment:
I don't think this is the right approach here, but you've highlighted an issue in the framework.

The exception classes are duplicates. I think we should import `wt.pf.CandidateNoteFound` into `cfme.exceptions`, and remove the local definition in `cfme.exceptions`.